ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-postsubmits.yaml

```diff
@@ -4,12 +4,12 @@ postsubmits:
     always_run: true
     branches:
     - ^master$
-    cluster: multi01
+    cluster: build10
```

Comment:
What is this?

Reply:
I have no idea, it was generated automatically by `make ci-operator-config jobs`.

Anyway, I had conflicts on the PR, after rebasing, and re-generating the job file, it disappeared.